pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -862,6 +864,11 @@ func (podGC *PodGC) GetLabelSelector() *metav1.LabelSelector {
 	return nil
 }
 
+// GetDelay gets the podGC delay as a duration.
+func (podGC *PodGC) GetDelay() time.Duration {
+	return time.Duration(podGC.DelaySeconds) * time.Second
```

Comment:
Will it handle the default? what will the delay be if the user didn't specify in Workflow?

Reply:
`DelaySeconds` should default to `0` and `GetDelay` in this case should return `0 seconds`, meaning the deletion will be queued right away.